tardis/visualization/widgets/shell_info.py

```diff
@@ -1,5 +1,9 @@
 import ipywidgets as ipw
 import pandas as pd
+import panel as pn
+
+# Initialize Panel extension
+# pn.extension('tabulator')
```

Comment:
Why is this commented out?

Reply:
https://github.com/tardis-sn/tardis/pull/3197/commits/6d5cafac1b8828b9f3549f1e18b09b706dd58a6e
I didn't want the extensions to modify how the logger widget behaves which is why I haven't yet done the automatic initialize extension right now. I will do that in the https://github.com/tardis-sn/tardis/pull/3123 if that's alright.

Reply:
Does this mean that these widgets will still need manual init until 3123 is merged?